### ISSUE TITLE
[BUG] Grid tool is not working properly on hex grid in v1.3.3

### ISSUE THREAD

**Issue by r-valyar** [mclemente/simplefog#12]
**Module Version:** v1.3.3.

**Describe the bug**
The Grid tool is no longer snapping to the actual grid, instead it is completely misaligned. The Grid tool was working fine before I upgrade Foundry. It seems the tool is not taking into account the actual grid type and size configured for the scene. The other tools work fine. 

There are no errors in the console. 

**To Reproduce**
1. Go to enable hex grid 
2. Click on Grid Tool
3. Attempt to reveal a hex
4. Snapping is incorrect (fog reveals fine though)

**Expected behavior**
To properly snap to the actual grid used in the scene.

**Screenshots**
I have colored the grid red and put measurement controls that properly snap to grid (default foundry). In addition, the grid settings for the map. 
![image](https://github.com/user-attachments/assets/046cf87b-6835-4992-9dff-2144bc64a0a1)
![image](https://github.com/user-attachments/assets/512778ef-7398-46fa-b671-6136bc3b359a)

**Browser:**
 - Firefox latest 130.0.1
 - MS Edge 130

**Foundry Version:** 
- v12.331
- 
**Game System:** 
Forbidden Lands v12.1.0

**Additional context**
No module that interacts with the grid are loaded, I also tried on clean world with just this module enabled.

**Comment by joshschmille** [comment]
I have run into exactly this same issue.

**Comment by mclemente** [comment]
I can't reproduce this issue. I've created a new world on the Forbidden Lands system and it works as expected.

Could you try creating a new world and test the module there?

**Comment by r-valyar** [comment]
Yes, when I create new world, import the core module the reveal is misaligned:

![image](https://github.com/user-attachments/assets/ae936c08-67b5-4248-a043-cc2b71edc7c4)

Would you like to send you the world for review?

**Comment by canemacchina** [comment]
Same issue here,  the fog revealing is not aligned with the grid. I'm using the same world as @r-valyar. That is forbidden land. If you want to try @mclemente you can just download the free module for forbidden land and create a new world using that module.

**Comment by designvoid** [comment]
I am having exact same issue with Forbidden Lands also - anyone have any solutions?

**Comment by LordoftheMorning** [comment]
I'm having this issue as well on my homebrew world, so this may not be specific to Forbidden Land. My homebrew world also uses a hex grid and simple fog for overworld exploration.

**Comment by mclemente** [comment]
Should be fixed now with version 1.5.2.

I believe the main issue was Simple Fog picking up the grid size early and never updating it if it ever gets changed in a session, and I assume things like Forbidden Lands used a different grid size than 100px.
